components/reviews/review-list.tsx

```
@@ -0,0 +1,95 @@
+
+  if (reviews.length === 0) {
+    return (
+      <div className="text-center py-8">
+        <p className="text-muted-foreground mb-4">
+          {userReviews
+            ? "You haven't submitted any reviews yet."
+            : "No reviews available for this provider yet."}
+        </p>
+        {userReviews && (
+          <Button asChild>
+            <a href="/providers">Browse Providers to Review</a>
```

ESLint

[error] Do not use an `<a>` element to navigate to `/providers/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)